foreman-selinux-disable

```diff
@@ -9,7 +9,7 @@ do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
     # Remove all user defined ports (including the default one)
     /usr/sbin/semanage port -E | \
-      grep elasticsearch_port_t | \
+      grep -E '(elasticsearch|docker)_port_t' | \
```

Comment:
I'd prefer to do it this way around if I may, because I'd probably include this in a 1.7 release.  And I've already started testing... :)